src/amoc.erl

```diff
@@ -25,7 +25,7 @@ do(Scenario, Count, Settings) ->
             case {amoc_controller:start_scenario(Scenario, Settings), Count} of
                 {ok, 0} -> ok;
                 {ok, Count} -> amoc_controller:add_users(1, Count);
-                Error -> Error
+                {Error, _} -> Error
```

Comment:
why? according to the spec it was correct before the change:
```
-spec start_scenario(amoc:scenario(), amoc_config:settings()) ->
ok | {error, term()}.
```

Reply:
`Error` was `{{error, Reason}, Count}`, which is not the spec. Here we want to return the error that `amoc_controller:start_scenario/2` returned.